torch/testing/_internal/common_utils.py

```diff
@@ -1981,11 +2021,12 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
             replace_with = torch.tensor(1, device=device, dtype=dtype)
         elif dtype in floating_types_and(torch.half, torch.bfloat16):
             replace_with = torch.tensor(torch.finfo(dtype).eps, device=device, dtype=dtype)
```

Comment:
Would you update this line, too, to use `torch.finfo(dtype).tiny` instead of `.eps`?